Makefile

```diff
@@ -8,8 +8,8 @@ SHELL=/bin/bash -o pipefail -o errexit
 
 AIO_VERSION       ?= 4.10.7-rc1
 CONTAINER_VERSION ?= test
-ALPINE_VERSION    = 3.18
-NGINX_VERSION     = 1.24.0
+ALPINE_VERSION    = 3.20
+NGINX_VERSION     = 1.26.1
```

Comment:
Please change the commit message as the version do not match

Now:

`NODE-5465 Bump versions: Alpine 3.20, Nginx 1.26.0`

Actually:

`NODE-5465 Bump versions: Alpine 3.20, Nginx 1.26.1`

Reply:
Did it myself